src/irsForms/ScheduleB.ts

```diff
@@ -0,0 +1,88 @@
+import { Income1099, Income1099Type, Information } from '../redux/data'
+import TaxPayer from '../redux/TaxPayer'
+import Form from './Form'
+import { computeField, displayNumber, sumFields } from './util'
+
+interface PayerAmount {
+  payer?: string
+  amount?: number
+}
+
+export default class ScheduleB implements Form {
+  state: Information
+
+  constructor (info: Information) {
+    this.state = info
+  }
+
+  f1099ints = (): Income1099[] =>
+    this.state.f1099s.filter((f) => f.formType === Income1099Type.INT)
+
+  l1 (): string[] {
+    const ints = this.f1099ints()
+
+    return (
+      Array
+        .from(Array(14))
+        .flatMap((v, i) => {
+          if (i < ints.length) {
+            return [ints[i].payer, ints[i].income.toString()]
+          }
+          return ['', '']
+        })
+    )
+  }
```

Comment:
Minor nitpick, I think this could be more clear as
```
return (
      Array
        .from(Array(14))
        .flatMap((_, i: number) => {
          if (i < ints.length) {
            return [ints[i].payer, ints[i].income.toString()]
          }
          return ['', '']
        })
    )
```

Reply:
Yea that could definitely be better. [See if you like the new diff](https://github.com/thegrims/UsTaxes/pull/118/commits/0cdb4e6d0ba90927682822e49169a8b7df665ede#diff-03642ce9739ea7e819879e4cfb46c2e76a31045ee41b6dbc44945da7a8b1a4f8R32)

Reply:
That's much more clear, thank you!